src/main/java/cam72cam/immersiverailroading/registry/EntityRollingStockDefinition.java

```diff
@@ -128,6 +129,11 @@ public void parseJson(JsonObject data) throws Exception  {
 		bogeyFront = (float) (data.get("trucks").getAsJsonObject().get("front").getAsFloat() * internal_model_scale);
 		bogeyRear = (float) (data.get("trucks").getAsJsonObject().get("rear").getAsFloat() * internal_model_scale);
 		
+		dampeningAmount = 0.5f;
```

Comment:
I played around with it a bit, 0.6 or 0.75 feels like a better default

Reply:
0.75 is now set as default